### PR TITLE
Fix testDeployUnsupportedKafka

### PR DESCRIPTION
**Pull request by stanlyDoge**
Signed-off-by: Stanislav Knot <sknot@redhat.com>

### Type of change
- Bugfix

### Description
This test was not even setting the non-existen kafka version.

### Checklist

- [ ] Update/write design documentation in `./design`
- [ ] Write tests
- [ ] Make sure all tests pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: systemtest/src/test/java/io/strimzi/systemtest/specific/SpecificST.java [hunk 1]
```diff
@@ -114,21 +114,15 @@ void testLoadBalancerIpOverride() throws Exception {
...
 
-        Condition condition = KafkaResource.kafkaClient().inNamespace(NAMESPACE).withName(CLUSTER_NAME).get().getStatus().getConditions().get(0);
-
-        verifyCRStatusCondition(
-                condition,
-                "Version 6.6.6 is not supported.",
-                "InvalidResourceException",
-                "True",
-                "NotReady");
+        assertThat(coLog.contains("Version " + nonExistingVersion + " is not supported"), is(true));
```
Comment (scholzj) [inline systemtest/src/test/java/io/strimzi/systemtest/specific/SpecificST.java]: Why did you removed the status check? It should IMHO be there. You are also not waiting for things to finish. So I wonder how do you know that it should be already in the log.

**Comment by stanlyDoge** [comment]
@strimzi-ci run tests profile=regression testcase=io.strimzi.systemtest.specific.SpecificST#testDeployUnsupportedKafka

**Comment by scholzj** [comment]
Would be good to get review / approval from @Seequick1 or @Frawless before merging this.

**Comment by Frawless** [comment]
This change will fix the test for sure.  I already have it in my fix changes locally.  I would like to have each part of kafka builder on separate line so we can merge this and I will "fix" in my changes or we can close it and this fix will be in my PR (I will try to open it today)

**Pull request merged**